mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -433,6 +433,14 @@ def process_code_coverage(key, value, timestamp):
                         self.logger.prn_inf("%s received"% (key))
                         callbacks__exit_event_queue = True
                         break
+                    elif key == '__timeout_set':
```

Comment:
The beginning of the test (the "preamble" as its referred to at the top of this file) sets the timeout of the test with the `__timeout` key. I'd say go ahead and reuse that key instead of making a new one, unless you have any objections.

Reply:
The `__timeout` key at "preamble" stage also sets the global timer to 0 and timeout duration. The behavior of `__timer_set` only sets the timeout duration, but doesn't change the global timer (e.g. time elapsed). I'd prefer not to confuse these two different behaviors.

Reply:
That's a very good point, thanks very much for clarifying 😄